README.md

```diff
@@ -27,6 +27,22 @@ npm install --save aws-appsync
 yarn add aws-appsync
 ```
 
+#### React Native Compatibility
+When using this library with React Native, you need to ensure you are using the correct version of the library based on your version of React Native. Take a look at the table below to determine what version to use.
+
+
+| `aws-appsync` version                     | Required React Native Version                                                 
+| ----------------------------------------- | -------------------------------------------------------------------------------- |
+| `2.x.x`                                   | `>= 0.60`
+| `1.x.x`                                   | `<= 0.59`                                                                      
+
+If you are using React Native `0.60` and above, you also need to install `@react-native-community/netinfo`:
+
+```
+npm install --save @react-native-community/netinfo
```

Comment:
Can we add a `yarn` option for install as well? e.g. 

```
npm ....
```
or
```
yarn add @react-native-community/netinfo
```

Reply:
Done!